includes/free/class-login.php

```diff
@@ -1091,4 +1079,62 @@ public function default_wp_login_override() {
             exit;
         }
     }
+
+    /**
+     * Send activated email for user
+     *
+     * @param $user
+     */
+    private function send_email_after_activation( $user ) {
```

Comment:
I've asked to name the function as `send_activation_email`, not sure why have you changed it?

Reply:
I've thought of activation as when it happens i.e when we sent an email for user account activation. but this email is triggered after activation when the admin approves the user. please advise me if i am doing wrong.

Reply:
Still, keep the name as I suggested.